DSCResourceCommonTests/DSCResourceCommonTests.psd1

```diff
@@ -0,0 +1,58 @@
+@{
+# Version number of this module.
+ModuleVersion = '1.10.0.0'
+
+# ID used to uniquely identify this module
+GUID = 'b3239f27-d7d3-4ae6-a5d2-d9a1c97d6ae4'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = '(c) 2013 Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Module with DSC Resources for CommonTests'
```

Comment:
This doesn't contain any DSC Resources, so I'd just call it something like "Common tests for DSC resources"